Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Bulk_Export_to_Cisco_ISE_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,29 @@
+This playbook gets all available device inventory from PANW IoT Cloud and updates/create endpoints with custom attributes on Cisco ISE.
```

Comment:
Gets the entire available device inventory from PANW IoT Cloud and updates/creates endpoints with custom attributes in Cisco ISE. You should run this playbook as a scheduled job.

{I would include an example of a job schedule and why, such as:
For example, you might want to schedule the job to run this playbook at the end of each day so that Cisco ISE is updated on a daily basis with the IoT device inventory.}